recipes/push_data.py

```diff
@@ -20,18 +21,27 @@ def run(args):
         from_date=args.from_date, to_date=args.to_date, branch=args.branch
     )
 
-    header = [
-        'Revision',
-        'All Tasks',
-        'Regressions (possible)',
-        'Regressions (likely)',
-    ]
+    if config.output_file is not None and os.path.exists(config.output_file):
```

Comment:
Maybe I'm missing something, but when would this ever be saved? Are you doing it manually?

I need to document it better, but you might want to use the `config.cache` object instead:
https://active-data-recipes.readthedocs.io/en/latest/usage.html#cache

While it was mainly meant to cache query results (which it does automatically), you can use it to cache any arbitrary results. For example:
https://github.com/ahal/ci-recipes/blob/master/recipes/scheduler_analysis.py#L83

Aside from automatically saving results, another benefit is that it'll automatically work with multiple backends, including a database or something like memcached/redis. So we could even share caches across users in the future.

Reply:
Done!